soccer/gameplay/tactics/stopped/circle_near_ball.py

```diff
@@ -131,7 +131,10 @@ def execute_completed(self):
                 num_robots += 1
 
         i = 0
-        for pt in self.get_circle_points(num_robots):
+        for pt in self.get_circle_points(6 if num_robots == 0 else num_robots):
```

Comment:
(in response to both comments)

I think it would be better to properly handle the case of having no robots in the play. We could either set a variable to tell that no robots are assigned, transition the state into 'waiting for robot' or just set none here. This feels wayy too hacky, and can easily break in the future :P